tests/Microsoft.Bot.Configuration.Tests/ConfigurationLoadAndSaveTests.cs

```diff
@@ -112,7 +112,9 @@ public async Task LoadFromFolderWithSecret()
             var secret = BotConfiguration.GenerateKey();
             var config = await BotConfiguration.LoadAsync(testBotFileName);
             await config.SaveAsAsync(OutputBotFileName, secret);
-            await BotConfiguration.LoadFromFolderAsync(".", secret);
+            
+            var config2 = await BotConfiguration.LoadFromFolderAsync(".", secret);
```

Comment:
Maybe `expectedConfig` instead of `config2`?, same for the rest of tests.

Reply:
we used config2 because that name is already used in another similar test asserts

Reply:
I'd change the name of the variable anyway.

Reply:
If there are changes made, maybe we can rename all the other variables as well for this case.

Reply:
renamed to `loadedConfig`